concrete-core-test/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "concrete-core-test"
-version = "0.1.0"
+version = "0.0.0"
```

Comment:
why the change here?

Reply:
Well, with this `0.0.0` version thing, I wanted to reflect the fact that this crate is actually not versioned, but rather, expected to be in full sync with `concrete-core` at any moment. 

This is valid for :
+ `concrete-core-fixture` 
+ `concrete-core-test` 
+ `concrete-core-bench`

Maybe we don't care actually :laughing: !

@agnesLeroy What's your opinion on that ?

Reply:
Good question ! 

We did it that way because it was difficult to architecture things nicely by using the conventional rust way for tests and benchmarks (which is to make special folders `tests` and `bench` in the package): 
+ We would have had to rely on compilation profiles to do things that are done easily with separate packages. 
+ We expect the test and bench application to become increasingly big and complex as the library grows (I wouldn't be surprised if we moved out from `criterion` and `libtest` in the future), so we thought it made sense to keep things separate in well separated crates.
+ It made it faster to develop those three guys, because we did not have to rebuild `concrete-core` every time we changed something in one of them (thanks to dependence caching). Which was great, given how long it is to compile fftw :sweat_smile:

Reply:
But the `0.0.0` idea is from me, this is not a rust good practice or anything I read elsewhere. I just thought it made it clear that there was something special about those guys.

Reply:
I really don't know at the moment, I did not have to care for compile times in a long time :p

Reply:
Hum, I'd say we can keep the `0.0.0` versions even if they seem a bit strange for now. The question raised by @IceTDrinker is interesting though: don't we want to have just one large crate with core/commons/npe/fixture/test/bench? It would be a little annoying for the tests because we'd have all the doc tests + the distribution tests to filter from, but other than that are there other cons?

Reply:
You already answered my question above actually @aPere3, so let's keep things like this with 0.0.0 version numbers. Thanks!

Reply:
I thought that maybe only what was needed would be compiled (like only benchmarks and all the deps for it), maybe through features ?

I really don't know enough about the build system to have a definitive answer at the moment

Reply:
Ok, for now, I'll just put `0.0.0` for all of those, and we may rework the topic later on ! Thanks for the remarks !